### PR TITLE
miscellaneous lint, clean-up and modernization changes to webdriver/tests/support

### PR DESCRIPTION
**Pull request by thiagowfx**

### CODE REVIEW

**Review by sadym-chromium — APPROVED** [review]

File: webdriver/tests/bidi/session/subscribe/invalid.py [hunk 1]
```diff
@@ -8,16 +8,16 @@
 
 
 @pytest.mark.asyncio
-async def test_params_empty(bidi_session, send_blocking_command):
+async def test_params_empty(send_blocking_command):
     with pytest.raises(InvalidArgumentException):
-        response = await send_blocking_command("session.subscribe", {})
```
Comment (whimboo) [inline webdriver/tests/bidi/session/subscribe/invalid.py:13]: Why do you assign the result to a variable that is not used? Same applies to other tests.

**Review by whimboo — CHANGES_REQUESTED** [review]

**Comment by whimboo** [comment]
@thiagowfx is that PR ready for review or are you still working on it? If it's the latter then please mark the PR as draft. Thanks.

**Comment by thiagowfx** [comment]
@whimboo it's done! I just added two small additional changes just now, but do not plan do add anything else.

**Comment by nechaev-chromium** [comment]
How can I find what tests are expected to fail for Chrome? For instance it is expected that "test_status_with_session_running_on_endpoint_node" will always fail. This expectation must be documented somewhere.

**Comment by thiagowfx** [comment]
@nechaev-chromium I wonder if this is the place where it should be documented: https://github.com/web-platform-tests/wpt-metadata/tree/master/webdriver/tests/classic

**Comment by whimboo** [comment]
> @whimboo it's done! I just added two small additional changes just now, but do not plan do add anything else.

I can still see more updates. So I want to ask again if it's done or if we can expect more changes. Thanks.

**Comment by whimboo** [comment]
Ok, so if you rebase then please do not force push but run the rebase via `git merge` so that a new commit will be added and it's visible for reviewers. With the huge number of commits it's hard to keep track off.

**Review by whimboo — APPROVED** [review]
Those updates look great. Would be good to have some more linting rules in the future as well so we don't end-up like that again. But that's another topic.

I had a look at the Firefox failures and here it's the merged https://github.com/web-platform-tests/wpt/pull/40122 which causes action tests to fail.

@jgraham I can you please admin merge this PR? Thanks.

**Comment by thiagowfx** [comment]
> Ok, so if you rebase then please do not force push but run the rebase via git merge so that a new commit will be added and it's visible for reviewers. 

I see, that's a good idea for large PRs like this one. Will keep this in mind for next time, thanks.

**Comment by thiagowfx** [comment]
I believe the only failure is due to https://github.com/web-platform-tests/wpt/issues/38450 (thus unrealted).

**Comment by whimboo** [comment]
Thank you @thiagowfx. We need @jgraham here to get this admin merged.

**Comment by thiagowfx** [comment]
> @thiagowfx this PR has a merge conflict now. Can you please resolve it?

Done. Out of habit I force pushed again, but will reiterate there are no modifications other than `response_completed.py` (removed some extraneous imports while resolving the merge conflict).

**Review by whimboo — APPROVED** [review]

**Pull request merged**